static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx

```diff
@@ -32,10 +33,13 @@ const TableView = () => {
           columns: [
             {
               content: "1",
-              role: "rowheader",
+              className: "u-text--muted",
             },
             {
-              content: <img src="http://placekitten.com/45/45" />,
+              content: (
+                <img src="https://assets.ubuntu.com/v1/9ef4a092-Architecture.svg" />
```

Comment:
Missing alt tag. What about we match what's already in the existing section? 
![image](https://user-images.githubusercontent.com/7452681/134336722-1ff5a405-79da-4807-916e-86484e5baa34.png)


```suggestion
                <img src="https://assets.ubuntu.com/v1/9ef4a092-Architecture.svg" alt="Badge for Ubuntu System Architecture" />
```

Reply:
I'd counter that suggestion with an empty alt tag - in this context I'm not sure the image adds anything critical that someone using assistive technology couldn't glean from the title of the module:

```suggestion
                <img src="https://assets.ubuntu.com/v1/9ef4a092-Architecture.svg" alt="" />
```

From [WebAIM](https://webaim.org/techniques/alttext/):

> If the content that the image conveys is presented within text in the surrounding context of the image, then an empty alt attribute may suffice.

Reply:
Yeah I know the content of the rows is not correct yet. I was planning to do that as part of this ticket : https://app.zenhub.com/workspaces/-commercial-squad-60910a7b9f3626000eff21c2/issues/canonical-web-and-design/commercial-squad/251

Reply:
@sowasred2012  Yup, empty alt tag totally works, too. As long as an image alt tag is handled in _some_ way.